components/backup/src/endpoint.rs

```diff
@@ -413,32 +425,69 @@ impl BackupRange {
         Ok(stat)
     }
 
-    fn backup_raw<E: Engine>(
+    fn is_value_valid(&self, value: &[u8]) -> bool {
+        if !self.is_raw_kv {
+            return true;
+        }
+        match_template_api_version!(
+            API,
+            match self.cur_api_version {
+                ApiVersion::API => {
+                    let raw_value = API::decode_raw_value(value).expect("invalid value");
```

Comment:
Suggest to return error other than panic. TiKV should not panic during backup.

Reply:
done